virt/virt-architecture.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="virt-architecture"]
+= {VirtProductName} architecture
+:context: virt-architecture
+
+toc::[]
+
+Learn about {VirtProductName} architecture.
+
+image::cnv_components_main.png[CNV Deployments]
```

Comment:
I looked at the CNV deployments diagram and compared it to the HCO-Operator section. 

If `deployment/hco-webhook` and `deployment/hyperconverged-cluster-cli-download` are components of the HCO Operator, then the logic for having them alongside the HCO Operator seems a little fuzzy to me. If the cnv_components_main.png diagram is correct, you might want to add a line explaining what a "deployment" is. Just a suggestion.